tex2wiki/README.md

```diff
@@ -1,6 +1,6 @@
 # tex2Wiki
 
-## Azeem Mohammed
+## Azeem Mohammed, Joon Bang
 
 WIP code
```

Comment:
It is a WIP
Again, this is a prototype still needs lots of work and tweaking, maybe some major rewrites of algorithms due to some issues I've encountered when testing with generated LaTeX code